dist/css/ember-widgets.css

```diff
@@ -45,6 +45,14 @@
   display: inline-block;
   outline: none;
 }
+.ember-select:focus .ember-select-choice,
```

Comment:
this is the compiled version of the changes in `select.less`, and he does need to check this in